core/src/utils/slot-mutation-controller.ts

```diff
@@ -50,6 +50,7 @@ export const createSlotMutationController = (
 
     hostMutationObserver.observe(el, {
       childList: true,
+      subtree: true,
```

Comment:
Let's add a comment explaining why this was adding. That way it's not accidentally removed in the future.